app/components/jobseekers/schools_overview_component.html.haml

```diff
@@ -68,11 +74,12 @@
                   = t('schools.address')
                 %td.govuk-table__cell
                   = full_address(organisation)
-              %tr.govuk-table__row
-                %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
-                  = t('schools.website')
-                %td.govuk-table__cell
-                  = link_to("#{organisation.name} website (opens in a new window)", organisation.url, class: 'govuk-link link-wrap', target: '_blank')
+              - if organisation.website.present? || organisation.url.present?
+                %tr.govuk-table__row
+                  %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
+                    = t('schools.website')
+                  %td.govuk-table__cell
+                    = link_to("#{organisation.name} website (opens in a new window)", organisation.website.presence || organisation.url, class: 'govuk-link link-wrap', target: '_blank')
```

Comment:
` website (opens in a new window)` is repeated a few times prior to this so would it better in en.yml?